.github/workflows/release.yaml

```diff
@@ -37,8 +27,8 @@ jobs:
 
   terraform-provider-release:
     name: 'Terraform Provider Release'
-    needs: [go-version, release-notes]
-    uses: hashicorp/ghaction-terraform-provider-release/.github/workflows/hashicorp.yml@v2
+    needs: [release-notes]
+    uses: manicminer/ghaction-terraform-provider-release/.github/workflows/hashicorp.yml@1dba3fa320b576c92d2206c9514e5f9622730010
```

Comment:
is there a PR tracking moving this back?

Reply:
Not yet, I'm trying to test this end-to-end first as I'm not sure that blindly passing in blank values will work with the eventual upstream action (it should in theory, but GHA keeps confounding me)